src/libraries/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/SyndicationElementExtensionCollectionTests.cs

```diff
@@ -206,8 +205,6 @@ public void ReadElementExtensions_EmptyExtensionName_ThrowsArgumentException()
         public void Clone_NoExtensions_ReturnsExpected()
         {
             var category = new SyndicationCategory();
-            SyndicationElementExtensionCollection elementExtensions = category.ElementExtensions;
-
```

Comment:
Rather than not checking the original extention collection, should there be an assertion that this collection is not empty?

Reply:
Yeah, you're correct! Though in this case I believe we want to assert that the collection _is_ empty?

I've made that change, let me know if it looks incorrect to you or if you have any questions. Thanks!